Wiki/starting/starting-leaving.md

```diff
@@ -0,0 +1,146 @@
+
+For the input files, some inputs are published data in which case you need to keep the reference to this data (including the version). If you have written several codes, the output of a piece of code will be the input of the next piece of code, in which case you do not necessarily need to keep that data,  but you need to describe your workflow.
+
+**Workflow**
+
+This can be a tricky one, as there is not a one-size-fits-all format to save this information. It is fine to write a README file and archive it with other files from the project that need archiving. You can also have a special Github repository just for this README file, or a repository for all your projects with READMEs for each project. Whatever format you choose, it is important for this information to be publicly available (unless your project was restricted) and not a personal note.
+
+This description should clearly describe step by step what someone should do to reproduce your work.
+
+```{warning}
+Information on your PhD or published paper has to be kept for at least 5 years. The time requirements differ slightly depending on institutions and funding bodies.
+```
+
+### To delete, always
+
+You do not need to keep files that are not necessary to reproduce your work:
+
+* log files
+* failed experiments
+* temporary files such as created from successive cdo/nco commands.
+
+```{tip}
+Occasionaly log files might have some information which is relevant to the workflow, still just keeping them makes this information virtually inaccessible to anyone else. The right thing to do in this case is to extract what is useful and add it to the workflow documentation, then getting rid of the files.
+```
+
+### To keep or to delete?
+Some of the data is harder to categorise, and need some careful consideration to decide if it needs to be kept or deleted. Neglecting to do so might result in the deletion of useful data or waste of storage
+
+**Model outputs**
+
+These are usually reproducible, even if it might not be possible to get bitwise reproducibility if underlying libraries change or the machine you have used is retired. 
+Such a decision should be weighed against the cost, in money and time, of reproducing your work.
+On the other side storing the data can also be expensive, as the model output are usually large,  and of little use if not done properly.
+That is where you need to discuss with your supervisor, and possibly other collaborators, as the answer might vary depending on how likely someone else might find this data useful.
+As model output includes raw output, restart files and post-processed output, they should all be treated differently. usually if keeping raw output this is stored on tape, it is a bit harder to retrieve but tape is less expensive than other storage. Only some of the restart files are kept (if any). Ideally, post-processed should be published if it has been used for publication or if it's going to be shared with others. 
```

Comment:
```suggestion
Discuss this matter with your supervisor, and possibly other collaborators, as the answer might vary depending on how likely someone else might find this data useful.
As model output includes raw output, restart files and post-processed output, they should all be treated differently. usually if keeping raw output this is stored on tape, it is a bit harder to retrieve but tape is less expensive than other storage. Only some of the restart files are kept (if any). Ideally, post-processed should be published if it has been used for publication or if it's going to be shared with others. 
```

Reply:
Only part of this has been committed ?? I fixed it in my branch